request_wrapper.py

```diff
@@ -0,0 +1,68 @@
+
+import subprocess
+import pycurl
```

Comment:
`pycurl` import should be placed after `json`, `os` and `StringIO` imports as pycurl is not a standard library (i.e. "from outside").
```
C:  4, 0: standard import "import json" should be placed before "import pycurl" (wrong-import-order)
C:  5, 0: standard import "from StringIO import StringIO" should be placed before "import pycurl" (wrong-import-order)
C:  6, 0: standard import "import os" should be placed before "import pycurl" (wrong-import-order)
```